### PR TITLE
LA-448, rename 'user_course_scores' view and add last_activity_at column

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/LA-448

I ran this locally; results are in `dev_johncrossman_boac` (data_loch_dev).

### CODE REVIEW

File: nessie/externals/redshift.py [hunk 1]
```diff
@@ -53,7 +53,8 @@ def execute_ddl_script(sql):
     statements = sql.split(';')
     # Remove any trailing debris after the last SQL statement.
     del statements[-1]
-    for statement in statements:
+    for index, statement in enumerate(statements):
+        app.logger.info(f'Executing DDL script {index + 1} of {len(statements)}')
```
Comment (johncrossman) [inline nessie/externals/redshift.py]: Logs progress. A bit reassuring when running long jobs.

File: nessie/sql_templates/create_boac_schema.template.sql [hunk 2]
```diff
@@ -200,15 +200,16 @@ AS (
         AND {redshift_schema_canvas}.submission_dim.workflow_state != 'deleted'
 );
 
-DROP TABLE IF EXISTS {redshift_schema_boac}.user_course_scores;
+DROP TABLE IF EXISTS {redshift_schema_boac}.course_enrollment;
 
-CREATE TABLE {redshift_schema_boac}.user_course_scores
+CREATE TABLE {redshift_schema_boac}.course_enrollment
```
Comment (johncrossman) [inline nessie/sql_templates/create_boac_schema.template.sql]: I suppose this should be plural based on naming convention. I'll amend to be `course_enrollments`

File: nessie/externals/redshift.py (hunk 1 above)
Comment (paulkerschen) [reply nessie/externals/redshift.py]: +1

**Comment by paulkerschen** [comment]
Looks good!

**Pull request merged**